### ISSUE TITLE
Doesn't work with multiple SoundCloud feeds

### ISSUE THREAD

**Issue by diegovogel** [OleVik/grav-plugin-twigfeeds#23]
I can't seem to get this to work with multiple feeds from SoundCloud. I think the issue may be that the RSS source files have the same name, even though they are from two unrelated podcasts. But I'm not sure about that and there would be nothing I could do about it anyway because I have no control over the source files.

Here's my Twig Feeds setup:
```
enabled: true
cache: true
static_cache: false
debug: false
cache_time: 900
pass_headers: false
silence_security: false
twig_feeds:
  -
    source: 'http://feeds.soundcloud.com/users/soundcloud:users:451210002/sounds.rss'
    name: 'The Call-Up'
    start: 0
    end: 2
  -
    source: 'http://feeds.soundcloud.com/users/soundcloud:users:398535258/sounds.rss'
    name: 'On Redemption'
    start: 0
    end: 2
```

And I'm testing by putting this in the content of a Grav page (with Twig processing turned on, etc):
```
<ul>

{% for name, feed in twig_feeds %}
    <li>{{name}}</li>
{% endfor %}

</ul>
```

I would expect to see a list with both of the names, but the list only has one name. I can get both of the names to show up if I replace either of the SoundCloud URLs with something else (like one of the feed URLs in the examples in the plugin documentation). And if I view the SoundCloud RSS content in the browser I don't see any issues. So I don't think there's anything wrong with either one of the SoundCloud URLs, just seems like they can't both be used by the plugin at the same time.

Any thoughts or workarounds?

**Comment by diegovogel** [comment]
@OleVik ? Anyone? Any guidance here would be much appreciated. This is a great plugin and I haven't found another one that does what this one does. Thanks!

**Comment by OleVik** [comment]
I can replicate this with your test-case, and the feeds both contain valid markup, but I cannot immediately see why iteration fails. It is not directly Twig-related, as dumping the `twig_feeds`-data only reveals a single item.

My current suspicion is with how the source-URLs are interpreted, that upon iteration the source may be interpreted as the same since the domain is the same, leading it to overwrite the second feed with the first.

**Comment by OleVik** [comment]
This should be fixed by [v3.3.2](https://github.com/OleVik/grav-plugin-twigfeeds/releases/tag/v3.3.2), your test-case works as expected with the change.

**Comment by diegovogel** [comment]
Awesome, thank you!
